xchange-core/src/main/java/org/knowm/xchange/dto/Order.java

```diff
@@ -224,10 +221,27 @@ public void setAveragePrice(BigDecimal averagePrice) {
 
     this.averagePrice = averagePrice;
   }
-
+  /**
+   * @deprecated CurrencyPair is a subtype of instrument <br>
+   *     use {@link #getInstrument()} instead like this:
+   *     <blockquote>
+   *     <pre>
+   * getInstrument()
+   * </pre>
+   *     </blockquote>
```

Comment:
change this javadoc to

```
/**
  * @deprecated
  * CurrencyPair is a subtype of Instrument - this method will throw
  * an exception if the order was for a derivative
  * <p> use {@link #getInstrument()} instead
  */
```

that example in blockquote is not needed

Reply:
updated